src/audio/copier/copier_host.c

```diff
@@ -0,0 +1,16 @@
+// SPDX-License-Identifier: BSD-3-Clause
+//
+// Copyright 2023 Intel Corporation. All rights reserved.
+//
+// Author: Baofeng Tian <baofeng.tian@intel.com>
+
+#include <sof/trace/trace.h>
+#include <sof/lib/uuid.h>
+
+LOG_MODULE_REGISTER(copier_host, CONFIG_SOF_LOG_LEVEL);
+
+/* af1f5122-f399-11ed-a05b-0242ac120003 */
+DECLARE_SOF_RT_UUID("copier_host", copier_host_comp_uuid, 0xaf1f5122, 0xf399, 0x11ed,
+		    0xa0, 0x5b, 0x02, 0x42, 0xac, 0x12, 0x00, 0x03);
+
+DECLARE_TR_CTX(copier_host_comp_tr, SOF_UUID(copier_host_comp_uuid), LOG_LEVEL_INFO);
```

Comment:
why are you declaring a new UUID @btian1 ?

Reply:
Previously, I thought it is a new module, ok, seems it is part of host, let me remove new UUID.